docs/components/operate/userguide/resolve-incidents-update-variables.md

```diff
@@ -119,6 +121,8 @@ We’ll publish a message that will be correlated with the instance, so we can a
 
 In the Operate interface, you should now observe the process instance has an <!-- FIXME: [“Incident”](/reference/incidents.html) --> incident, which means there’s a problem with process execution that must be fixed before the process instance can progress to the next step.
 
+<!--- What is the FIXME we have going on here? --->
```

Comment:
It looks like it is supposed to reference https://docs.camunda.io/docs/components/concepts/incidents/ perhaps?